cacti/datadog_checks/cacti/cacti.py

```diff
@@ -95,7 +97,7 @@ def _get_config(self, instance):
         password = instance.get('mysql_password', '') or ''
         db = instance.get('mysql_db', 'cacti')
         rrd_path = instance.get('rrd_path')
-        whitelist = instance.get('rrd_whitelist')
+        whitelist = instance.get('rrd_whitelist', None)
```

Comment:
Is this to take care of the case of `rrd_whitelist` being set to a "falsy" value like an empty string or False? This should be `instance.get(rrd_whitelist) or None` then

Reply:
This change is actually unnecesary